src/engine/BMInterface.php

```diff
@@ -605,6 +613,38 @@ protected function retrieve_button_ids($playerIdArray, $buttonNameArray) {
         return $buttonIdArray;
     }
 
+    public function save_join_game_decision($playerId, $gameId, $decision) {
+        if (!is_bool($decision)) {
+            throw new InvalidArgumentException('decision must be boolean');
+        }
```

Comment:
Editorial comment: for the most part, the other public BMInterface() functions take text arguments like 'add' or 'decline'.  It seems weird that this one should take a boolean, and, given the little API inconsistencies we already have with arguments which are passed as booleans (there are a couple of outstanding technical debt items about this), i'm lukewarm on booleans as being a simple interface.  I think we should expect that the API function for whether or not to join a game will pass a string, and that apiResponder.php will pass that string on to BMInterface as a string.